crates/bindings-csharp/Codegen/Module.cs

```diff
@@ -310,38 +363,36 @@ public ReducerDeclaration(GeneratorAttributeSyntaxContext context)
         {
             throw new Exception($"Reducer {method} must return void");
         }
+        if (method.Parameters.FirstOrDefault()?.Type is not INamedTypeSymbol namedType || namedType.Name != "ReducerContext") {
```

Comment:
Please check the full type name via `.ToString()` or `.FullName`, to make sure it's _our_ `ReducerContext`, just in case.

Reply:
Hrm the reason I did this cast to INamedTypeSymbol is because `ReducerContext` doesn't exist. Trying to make the ISymbol resolve the type its supposed to point to causes a generator exception.

Ours is `BaseReducerContext`, so there shouldn't be any conflicts possible.

Reply:
Or rather, this very codegen pass is what generates `ReducerContext` now

Reply:
Oh right, because we autogenerate it... That's unfortunate, but makes sense.